javalin/src/main/java/io/javalin/http/JavalinServletHandler.kt

```diff
@@ -109,19 +102,11 @@ class JavalinServletHandler(
                     exceptionMapper.handleFutureException(ctx, throwable)
                 }
             }
-            ?.thenAccept { if (it.result.future != null) tasks.offerFirst(createFutureCallbackTask(it)) } // add future callback add the beginning of queue
+            ?.thenAccept { if (it.result.future != null) tasks.offerFirst(createUserCodeTask()) } // add future callback add the beginning of queue
             ?: completedFuture(null) // stub future if task was skipped
 
-    @Suppress("UNCHECKED_CAST")
-    private fun createFutureCallbackTask(executionResult: ExecutionResult): Task =
-        Task(
-            stage = FutureCallbackStage.stage,
-            handler = {
-                executionResult.result.callback
-                    ?.let { (it as Consumer<Any?>).accept(executionResult.value) }
-                    ?: ctx.contextResolver().defaultFutureCallback(ctx, executionResult.value)
-            }
-        )
+    private fun createUserCodeTask(): Task =
```

Comment:
I don't think it's needed at all, it appends new scope with empty lambda, so it literally does nothing.

Reply:
By removing it you get 

```java.lang.IllegalStateException: Cannot override result```:

```
@Test
fun `calling future in (before - get - after) handlers works`() = TestUtil.test { app, http ->
    app.before("/future") { it.future(getFuture("before").thenAccept { v -> it.result(v) }) }
    app.get("/future") { it.future(getFuture("nothing")) }
                              ^^^^^ HERE ^^^^^ 
    assertThat(http.get("/future").body).isEqualTo("before, after")
}
```